source/whats-new.txt

```diff
@@ -0,0 +1,57 @@
+==========
+What's New
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Learn what's new in:
+
+* :ref:`Version 4.2 <version-4.2>`
+* :ref:`Version 4.1 <version-4.1>`
+* :ref:`Version 4.0 <version-4.0>`
+
+
+.. _version-4.2:
+
+What's New in 4.2
+-----------------
+New features of the 4.2 Java driver release include:
+
+- Added Azure and GCP key stores to client-side field level encryption
+- Added Kerberos caching tickets for reuse in multiple authentication requests
+- Added `com.mongodb.MongoClient <http://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-sync/com/mongodb/client/MongoClients.html>`_ instances with ``MongoClientSettings`` or ``ConnectionString`` as the configuration
```

Comment:
```suggestion
- Added `MongoClients <http://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-sync/com/mongodb/client/MongoClients.html>`_ instances with ``MongoClientSettings`` or ``ConnectionString`` as the configuration
```

Any reason to link to MongoClients but not link to the other items that appear to have been added? Additionally, are these new classes?

Reply:
no, it existed since 3.7, but it's new as a parameter to pass into the constructor